oat-sa/tao-core
# Feature/CON-76/advance search in result page

Related to: https://oat-sa.atlassian.net/browse/CON-76
https://github.com/oat-sa/extension-tao-outcomeui/pull/357
https://github.com/oat-sa/lib-tao-elasticsearch/pull/40
- [ ] https://github.com/oat-sa/tao-core-ui-fe/pull/222

FE part: changes in `views/js/layout/search.js`:

1. search sends to `searchModal`  hideResourceSelector: true id it is Results page, and placeholder
2. after selecting search result store` delivery result id` in `searchComponent.container`

How to test:
- go to any page and check that placeholder for search input `Search Item`, `Search Result` used also in searchModal
- go to Results page
- run search (use delivery name or test taker name)
- in searchModal should be hidden `add criteria` button and resource selector
- check table with search results and press `Go To Item`
- delivery result should be loaded
